tutorials/filter/README.md

```diff
@@ -0,0 +1,7 @@
+Sample application to show how to use the filter operator.
```

Comment:
Please put details instructions for how to launch compile and launch the application etc.
Also if any property have to be changed by user, please specify that explicitly.